src/components/ArticleComment/ArticleComment.module.scss

```diff
@@ -0,0 +1,3 @@
+.comment-text {
+color: #686767;
+}
```

Comment:
You might want to add a new line here, to avoid the error flags from Github. Please do the same for the other `.scss` files too

Reply:
Other than the comment above, all others look good to me.

Reply:
Thanks @d-beloved I'll fix that.